pkg/api/article.go

```diff
@@ -134,25 +139,27 @@ func CreateArticle(baseUrl string, apiKey string, apiSecret string, channelId st
 	)
 
 	if err != nil {
-		log.Fatal(err)
+		return err
 	}
 
 	client := &http.Client{}
 
 	resp, err := client.Do(req)
 	if err != nil {
-		log.Fatal(err)
+		return err
 	}
 	defer resp.Body.Close()
```

Comment:
I guess we can remove these since you return the close right?

Reply:
we still want it to close when line 149 returns err